addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.xml

```diff
@@ -62,4 +49,40 @@
             </t>
         </div>
     </div>
+
+    <div t-name="spreadsheet_dashboard.DashboardAction.Collapsed">
+        <div class="bg-view h-100 o_search_panel_sidebar cursor-pointer" t-on-click="toggleSidebar">
+            <div class="d-flex">
+                <button class="btn btn-light btn-sm m-1 mb-2 p-2">
+                    <i class="fa fa-fw fa-angle-double-right"/>
```

Comment:
```suggestion
                <button class="btn btn-sm m-1 mb-2 p-2">
                    <i class="fa fa-angle-double-right"/>
```
these classes don't seems to do anything

Reply:
fa-fw is indeed useless but btn-light actually affects the button backgorund when hovered ;-)